examples/active-class-name/components/Nav.js

```diff
@@ -22,6 +22,11 @@ const Nav = () => (
           <a className="nav-link">About</a>
         </ActiveLink>
       </li>
+      <li>
+        <ActiveLink activeClassName="active" href="/[slug]" as="/slug">
+          <a className="nav-link">Slug</a>
```

Comment:
This feels less confusing, as `/slug` is very similar to `/[slug]`, and `Dynamic Route` gives you a better idea of what the link means. 

```suggestion
        <ActiveLink activeClassName="active" href="/[slug]" as="/dynamic-route">
          <a className="nav-link">Dynamic Route</a>
```

Reply:
it is the example... it is not dynamic.. may be static

Reply:
maybe better would be `/some-slug` ?

Reply:
dynamic entails non static routes...

Reply:
@AubreyHewes It's static but also dynamic, with Next.js you can have both!

Reply:
users who use a `[slug].ts` or `[...slug].ts` possibility will understand the example... i.e. that you can do a `NavLink` on the `as` as well as the proposed `href` match.. cms pages (getStaticPaths) vs hardcoded pages. Anyway If you think that it is required to rename the path as `Dynamic Route` that is fine, I just do not agree.